Sources/Plasma/CoreLib/HeadSpin.h

```diff
@@ -411,7 +411,7 @@ void DebugMsg(const char* fmt, ...);
 
 #if defined(_MSC_VER)
 #define  DEFAULT_FATAL(var)  default: FATAL("No valid case for switch variable '" #var "'"); __assume(0); break;
-#elif defined(__GNUC__)
+#elif defined(__GNUC__) || defined(__clang__)
```

Comment:
Hmm, is that accurate?  I would have expected this to be more of a `clang` vs `clang-cl` distinction, not a Unix vs Windows distinction...

Reply:
Yeah, that's what I would have expected too, but it seems to make no difference. I tested using the Clang 12 from VS 2019, and both `clang` and `clang-cl` define only `__clang__` and not `__GNUC__`.

Reply:
Ok, it might be more of an SDK thing then -- Clang from MSYS2, for example, does define `__GNUC__` even on Windows, but that version would be using the MinGW SDK rather than the MSVC SDK.  Either way, the fix is appropriate, since Clang doesn't always define `__GNUC__`...  It just piqued my curiosity :)